### PR TITLE
Add ci-cluster-pool/ci-ocp-4-12-amd64-aws-us-east-1

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://redhat-internal.slack.com/archives/CBN38N3MW/p1674764037473279

/cc @openshift/test-platform

### CODE REVIEW

File: clusters/hive/pools/openshift-ci/ci-ocp-4-12-0-amd64-aws-us-east-1_clusterpool.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    version_lower: 4.12.0-0
+    version_upper: 4.13.0-0
+  name: ci-ocp-4-12-amd64-aws-us-east-1
+  namespace: ci-cluster-pool
+spec:
+  baseDomain: hive.aws.ci.openshift.org
+  hibernationConfig:
+    resumeTimeout: 20m0s
+  imageSetRef:
+    name: ocp-release-4.12.1-x86-64-for-4.12.0-0-to-4.13.0-0
```
Comment (danilo-gemoli) [inline clusters/hive/pools/openshift-ci/ci-ocp-4-12-0-amd64-aws-us-east-1_clusterpool.yaml:21]: I see `ocp-release-4.12.1-x86-64-for-4.12.0-0-to-4.13.0-0` configured nowhere as of now. Do we have to wait for `clusterimageset-updater` to sort things out, right?

Comment (hongkailiu) [reply clusters/hive/pools/openshift-ci/ci-ocp-4-12-0-amd64-aws-us-east-1_clusterpool.yaml:21]: I believe we have automation to refresh it.

Comment (danilo-gemoli) [reply clusters/hive/pools/openshift-ci/ci-ocp-4-12-0-amd64-aws-us-east-1_clusterpool.yaml:21]: Yeah, just wanted to be sure and have your opinion.

**Pull request merged**